scenarios/kubernetes_e2e.py

```diff
@@ -36,7 +36,7 @@
 ORIG_CWD = os.getcwd()  # Checkout changes cwd
 
 # Note: This variable is managed by experiment/bump_e2e_image.sh.
```

Comment:
not so experimental anymore, is it? :-)

Reply:
yeah I'll move it out to somewhere. @zmerlynn your feature graduated from alpha to beta now!